source/_posts/2019-03-23-using-symfony-webpack-encore-with-sculpin.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Using Symfony Webpack Encore with Sculpin"
+author:
+    shortname: Rae
+    name: Rae Knowler
+    twitter: RaeKnowler
+---
+
+The [Sculpin Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton) is a template for creating a blog with Sculpin. It's used in the [Get
+Started](https://sculpin.io/getstarted/) tutorial on sculpin.io, making it an important part of the Sculpin experience for many users. Until recently,
+the Blog Skeleton handled JavaScript assets by installing components via Composer. It's now been updated to use Symfony's
+[Webpack Encore](https://symfony.com/doc/current/frontend.html) library instead.
+
+Three of us at [Liip](https://www.liip.ch/en), [Christian Riesen](https://twitter.com/christianriesen), [David Buchmann](https://twitter.com/dbu) and
+I, had a hackday to work on Sculpin. [Kevin Boyd](https://twitter.com/beryllium9) offered us some suggestions of where to start, one of which was the
+move to Webpack: a more modern way of handling JS and CSS assets.
+
+Of course, you *can* make a website without any JS at all, but even a simple website needs some styling to look nice. The Blog Skeleton, which isn't
+meant to look fancy, uses jQuery, Bootstrap and HighlightJS as well as a little custom CSS. With more dependencies, keeping everything organised can
+get difficult. Encore takes all your CSS and JS files, as well as their dependencies, and bundles them up into one of each that you can link
+to in your HTML (or Twig) templates.
+
+It was simple to install Encore and configure it to work with Sculpin's file structure. Now, anyone building their blog using the Blog Skeleton as a
```

Comment:
the "simple" can be deterring for people trying to do this and not figuring out how to do it (see "belitteling language"). maybe instead say "You can see the changes that where necessary to install Encore and configure it to work with Sculpin's file structure in [this blogpost](link-to-blogpost)." - that would also help people who want to do this themselves.